sympy/polys/tests/test_numberfields.py

```diff
@@ -545,6 +545,7 @@ def test_AlgebraicNumber():
     assert a.is_aliased is True
 
     assert AlgebraicNumber(sqrt(2), []).rep == DMP([], QQ)
+    assert AlgebraicNumber(sqrt(2), (0, 0)).rep == DMP([], QQ)
```

Comment:
Thanks. Your are right. I think I see the difference now.
```
In [18]: p = ()

In [19]: p != Tuple()
Out[19]: False

In [20]: q = []

In [21]: q != Tuple()
Out[21]: True
```
So ``q = AlgebraicNumber(sqrt(2), ())`` was, in fact, going to the default branch leading to ``sqrt(2)`` instead of 0. I shall add a test for that.